src/setup_types.jl

```diff
@@ -35,3 +35,13 @@ function fill_wrapped_types!(wrapped_types_dict, function_type_list)
 end
 
 fill_wrapped_types!(TypeConversionFunctions, get_type_names())
+
+# libcxxwrap-julia prior to 0.8 mapped C++ copy to Base.deepcopy
+# now it is mapped to Base.copy
+# we make sure both really do a C++ copy for polymake types
+# whether this is deep or not cannot be enforced anyway
+if current_jlcxx >= v"0.8.0"
```

Comment:
We already have the global constant `jlcxx_version` which has to match `current_jlcxx` otherwise, the `check_jlcxx_version` errors out.

 ```suggestion
if jlcxx_version >= v"0.8.0"
```